README.md

```diff
@@ -182,22 +177,6 @@ Feature flags are used to show or hide certain features.
 
 - they are defined in the JSON config files within the `config` directory
 - specify an environment variable which can be used to override the setting in `config/custom-environment-variables`
-- these env vars are then configured using terraform, set the variable to the desired value in the `[env].tfvars` representing the desired environment
```

Comment:
these instruction should still be there as you still have `port MYA_FEATURE_FLAG=true` ? and in case you anything new feature flag this will still be good ?